.vscode/settings.json

```diff
@@ -1,6 +1,11 @@
 {
-    "css.lint.unknownAtRules": "ignore",
-    "css.customData": [
-        ".vscode/tailwind.json"
-    ]
-}
\ No newline at end of file
+  "css.lint.unknownAtRules": "ignore",
+  "css.customData": [".vscode/tailwind.json"],
```

Comment:
When you get a chance @iskay can you update this file with your VScode settings from your settings.json. It will help homogenize our editor settings so we don't just push spacing changes (like I've already done :D)